Tests/Unit/Command/DescribeCommandTest.php

```diff
@@ -126,4 +134,175 @@ public function execute()
+        $commandTester->execute([
+            'deploymentName' => $this->deployment->getName(),
+        ]);
+        $this->assertEquals(
+            '<success>Deployment TestDeployment</success>
+
+Workflow: <success>Simple workflow</success>
+
+Nodes:
+
+  <success>TestNode</success> (hostname)
+
+Applications:
+
+  <success>TYPO3 CMS:</success>
+    Deployment path: <success></success>
+    Options: 
+      packageMethod => <success>git</success>
+      transferMethod => <success>rsync</success>
+      updateMethod => <success>NULL</success>
+      context => <success>Production</success>
+      scriptFileName => <success>vendor/bin/typo3cms</success>
+      webDirectory => <success>web</success>
+      rsyncExcludes =>
+        <success>.git</success>
+        <success>web/fileadmin</success>
+        <success>web/uploads</success>
+      TYPO3\Surf\Task\Generic\CreateDirectoriesTask[directories] =>
+      TYPO3\Surf\Task\Generic\CreateSymlinksTask[symlinks] =>
+      deploymentPath => <success>NULL</success>
+      releasesPath => <success>/releases</success>
+      sharedPath => <success>/shared</success>
+    Nodes: <success>TestNode</success>
+    Detailed workflow: 
+      initialize:
+        tasks:
+          <success>TYPO3\Surf\Task\CreateDirectoriesTask</success> (for application TYPO3 CMS)
+          <success>Task TYPO3\Surf\Task\Generic\CreateDirectoriesTask after TYPO3\Surf\Task\CreateDirectoriesTask</success> (for application TYPO3 CMS)
+      package:
+        tasks:
+          <success>TYPO3\Surf\Task\Package\GitTask</success> (for application TYPO3 CMS)
+          <success>Task TYPO3\Surf\DefinedTask\Composer\LocalInstallTask after TYPO3\Surf\Task\Package\GitTask</success> (for application TYPO3 CMS)
+      transfer:
+        tasks:
+          <success>TYPO3\Surf\Task\Transfer\RsyncTask</success> (for application TYPO3 CMS)
+        after:
+          <success>TYPO3\Surf\Task\Generic\CreateSymlinksTask</success> (for application TYPO3 CMS)
+          <success>TYPO3\Surf\Task\TYPO3\CMS\CreatePackageStatesTask</success> (for application TYPO3 CMS)
+      update:
+        after:
+          <success>TYPO3\Surf\Task\TYPO3\CMS\SymlinkDataTask</success> (for application TYPO3 CMS)
+          <success>TYPO3\Surf\Task\TYPO3\CMS\CopyConfigurationTask</success> (for application TYPO3 CMS)
+      migrate:
+        tasks:
+          <success>TYPO3\Surf\Task\TYPO3\CMS\SetUpExtensionsTask</success> (for application TYPO3 CMS)
+      finalize:
+      test:
+      switch:
+        tasks:
+          <success>TYPO3\Surf\Task\SymlinkReleaseTask</success> (for application TYPO3 CMS)
+        after:
+          <success>TYPO3\Surf\Task\TYPO3\CMS\FlushCachesTask</success> (for application TYPO3 CMS)
+      cleanup:
+        tasks:
+          <success>TYPO3\Surf\Task\CleanupReleasesTask</success> (for application TYPO3 CMS)
+', $commandTester->getDisplay()
+        );
+    }
+
+    /**
+     * @test
+     */
+    public function describeNeosNeos()
```

Comment:
NeosNeos -> only Neos

Reply:
To have it streamlined with the composer package and namespace... \TYPO3\Surf\Application\Neos\Neos, \TYPO3\Surf\Application\Neos\Flow...

Reply:
I see.